controllers/flowlogspipeline/flp_objects.go

```diff
@@ -243,6 +245,13 @@ func (b *builder) addTransformStages(lastStage *config.PipelineBuilderStage) {
 		lokiWrite.URL = b.desiredLoki.URL
 		lokiWrite.TimestampLabel = "TimeFlowEndMs"
 		lokiWrite.TimestampScale = "1ms"
+		lokiWrite.TenantID = b.desiredLoki.TenantID
+		//TODO: set proper tls config https://issues.redhat.com/browse/NETOBSERV-309
+		lokiWrite.ClientConfig = &promConfig.HTTPClientConfig{
```

Comment:
Is there an authentication that happens with distributor/ingester when we use loki-operator without gateway mode?

Reply:
Loki doesn't manage authentication. It only force you to send `TenantID` when using TLS. That's why the gateway is a custom component :smile_cat: